History.txt

```diff
@@ -1,5 +1,30 @@
 Documentation and download available at https://www.FreeRTOS.org/
 
+Changes between FreeRTOS V10.4.3 LTS Patch 2 and FreeRTOS V10.4.3 LTS Patch 3 released September 16 2022
+
+	+ ARMv7-M and ARMv8-M MPU ports: It is possible for a third party that

```

Comment:
The history file should make it clear these issues applied to the prior version. So "it was possible", rather than "is possible", as a minimum.